bootstrap/mix.exs

```diff
@@ -3,7 +3,7 @@ defmodule Nerves.Bootstrap.Mixfile do
 
   def project do
     [app: :nerves_bootstrap,
-     version: "0.3.0-dev",
+     version: "0.3.0",
      elixir: "~> 1.4",
```

Comment:
okay, but if all this lock down gets weird, I am totally going to refer back to this conv 😺